new/LIT6205Saalnaka.xml

```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6205Saalnaka" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="gez" xml:id="t1">ሰአልናከ፡ መሐሪ፡ ሰአልናከ፡ ፈጣሪ፡ ሰዓልናከ፡ ከሀሊ፡ </title>
+              <title xml:lang="gez" corresp="#t1">Saʾalnāka maḥari saʾalnāka faṭāri saʾalnāka kahāli </title>
+              <title xml:lang="en" corresp="#t1">We have asked you, O Merciful One, we have asked you, O Creator...</title>
+
+              <title xml:lang="en" xml:id="t2">We have asked thee, O Merciful One, we have asked thee, O Creator, we have asked
+                 thee, O Mighty One...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```

Comment:
Could you maybe add a short abstract in this and the other record to clarify their division? I know that there is already a relation, but abstracts are very user-friendly.